vm/src/stdlib/os.rs

```diff
@@ -260,7 +260,7 @@ pub fn errno_err(vm: &VirtualMachine) -> PyBaseExceptionRef {
 #[allow(dead_code)]
 #[derive(FromArgs, Default)]
 pub struct TargetIsDirectory {
-    #[pyarg(named, default = "false")]
+    #[pyarg(positional, default = "false")]
```

Comment:
I think this should actually be `any` - it looks like it can accept `target_is_directory=` as a keyword argument too.

Reply:
Yes it should be `any` and now it is! Thanks for catching this.